package.json

```diff
@@ -50,13 +53,19 @@
     "ml-matrix": "^6.8.0"
   },
   "devDependencies": {
-    "@babel/plugin-transform-modules-commonjs": "^7.16.0",
+    "@types/jest": "^27.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.3.0",
+    "@typescript-eslint/parser": "^5.3.0",
     "eslint": "^8.1.0",
-    "eslint-config-cheminfo": "^7.2.0",
+    "eslint-config-cheminfo-typescript": "^10.2.2",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jest": "^25.2.2",
```

Comment:
We expect npm7+ to be installed (it comes by default with Node.js 16 and 17). With it, the peer dependencies are installed automatically